gobblin-data-management/src/main/java/gobblin/data/management/convertion/hive/LongWatermark.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.data.management.convertion.hive;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+import gobblin.source.extractor.Watermark;
+
+/**
+ * A {@link Watermark} that uses a single long value as watermark.
+ */
+@Getter
+@AllArgsConstructor
+public class LongWatermark implements Watermark, Comparable<LongWatermark> {
```

Comment:
Can you either rename this class, or use [this one](https://github.com/zliu41/gobblin/blob/sf/gobblin-core/src/main/java/gobblin/source/extractor/extract/LongWatermark.java)?

Because this watermark class doesn't support incrementing the watermark value, and always returns 0 for `calculatePercentCompletion`, which is counter-intuitive to be called `LongWatermark`.